modules/nf-core/raxmlng/tests/main.nf.test.snap

```diff
@@ -0,0 +1,139 @@
+{
+    "sarscov2 - bootstrap - stub": {
+        "content": [
+            {
+                "0": [
+                    [
+                        {
+                            "id": "test_stub"
+                        },
+                        "test_stub.raxml.bestTree:md5,d41d8cd98f00b204e9800998ecf8427e"
+                    ]
+                ],
+                "1": [
+                    
+                ],
+                "2": [
+                    "versions.yml:md5,193f5ef4cf09a15508291604fac6fa8e"
+                ],
+                "phylogeny": [
+                    [
+                        {
+                            "id": "test_stub"
+                        },
+                        "test_stub.raxml.bestTree:md5,d41d8cd98f00b204e9800998ecf8427e"
+                    ]
+                ],
+                "phylogeny_bootstrapped": [
+                    
```

Comment:
Why is it empty ?

Reply:
Excellent catch, because args wasn't in the stub.